config/default.yml

```diff
@@ -1,5 +1,6 @@
 AllCops:
   NewCops: disable
+  SuggestExtensions: false
```

Comment:
@kevindew so I noticed you saw the mintiest suggestion in some repos. I don't think we shouldn't be enabling this option here, instead it should be added to the local project rubocop config as it feels like project specific annoyance. Possible that projects (in and outside GOV.UK) use this gem and extension suggestions would indeed be helpful. 

This also has a larger impact of silencing not just suggestions for minitest, but all extension suggestions. I feel that at the very least, we could limit it to that:
```
AllCops:
  SuggestExtensions:
    rubocop-minitest: false
```

Reply:
Sure, I see the problem this option faces us with is one of consistency rather than it being a project annoyance, where the suggestion encourages you to take an approach inconsistent with the direction of this project. rubocop-govuk acts as a batteries included tool with various extensions and configs bundled that you can choose, and across GOV.UK there is a favouring (and lots of work) into trying to keep apps consistent with this styleguide rather than deviating in their own directions. This suggestion seems to encourage an app to go in its own direction and to end up with some extensions provided by this gem and others at an app level which seems potentially harmful.

I wouldn't have qualms with extra extensions, that are useful, being a part of this gem - minitest seems reasonable given a few of our apps use it. But I think having rubocop-govuk suggest ones to install at a project level is a step away from consistency.

Reply:
Yup, I also agree we should turn off suggestions: in addition to what @kevindew says, we're not acting on the ones we have, so it's not clear what the benefit of the suggestion is. We can always turn it on in future if we feel we've missed out on something in the majority of our repos.